### ISSUE TITLE
How to control the number of threads

### ISSUE THREAD

**Issue by yitengfei120011** [broadinstitute/picard#1950]
hi, How do I control the number of threads of MarkDuplicates when I have a large sample size?

**Comment by kockan** [comment]
I don't believe there is a way to specify the number of threads for MarkDuplicates. There should be some multi-threaded garbage collection / asynchronous I/O going on in the background, but the program itself isn't multi-threaded. GATK has a Spark implementation (https://gatk.broadinstitute.org/hc/en-us/articles/21904999597979-MarkDuplicatesSpark) that might make a difference in terms of performance for your case.
